libmscore/edit.cpp

```diff
@@ -3609,7 +3613,7 @@ void Score::undoChangeFretting(Note* note, int pitch, int string, int fret, int
 //   undoChangeKeySig
 //---------------------------------------------------------
 
-void Score::undoChangeKeySig(Staff* ostaff, const Fraction& tick, KeySigEvent key)
+void Score::undoChangeKeySig(Staff* ostaff, const Fraction& tick, KeySigEvent key, InstrumentChange* ic)
```

Comment:
it's worth adding a note here to make it easier to understand this function's behaviour, like so:

    ..., InstrumentChange* ic /* = nullptr */)

instead of just 

    ..., InstrumentChange* ic)